torch/_dynamo/allowed_functions.py

```diff
@@ -233,6 +236,17 @@ def _find_torch_objects(module):
     _find_torch_objects(torch)
     _find_torch_objects(math)
 
+    if config.trace_distributed:
```

Comment:
do you have to import torch.distribute._fun... here?

Reply:
It seems like we do not.